justfile

```diff
@@ -0,0 +1,43 @@
+# Define variables
+set dotenv-required
+
+DB_CONTAINER_NAME := "mina-mesh-db"
+
+# Download archive DB dump
+dl network:
+    ./tasks/dl.sh --network {{network}}
+
+# Initialize PostgreSQL database
+pg-init:
+    docker run -d --name {{DB_CONTAINER_NAME}} -p 5432:5432 -v $(pwd)/sql_scripts:/docker-entrypoint-initdb.d -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_USER=$POSTGRES_USER postgres postgres -c log_statement=all -c log_min_duration_statement=0
```

Comment:
I'm thinking also, that we might actually define `$PG_PORT` in `.env.example` too (and re-use it here as `-p $PG_PORT:5432`). 
I locally have a postgres running on `5432` and if I try to init pg docker environment I get a port conflict... so I need to manually change all the ports in my `.env` and here in order to make it work.